src/lib/list/selection-list.spec.ts

```diff
@@ -0,0 +1,299 @@
+import {async, TestBed} from '@angular/core/testing';
+import {Component} from '@angular/core';
+import {By} from '@angular/platform-browser';
+import {MdSelectionList, MdListOption, MdListModule} from './index';
+import {createKeyboardEvent} from '@angular/cdk/testing';
+import {UP_ARROW, DOWN_ARROW, SPACE} from '../core/keyboard/keycodes';
+import {Platform} from '../core/platform/index';
+
+
+describe('SelectionList and ListOption', () => {
```

Comment:
Need tests for `aria-selected` and `aria-disabled` (can be extra checks added to existing selection tests)